tests/unit/test_vcs.py

```diff
@@ -366,3 +376,50 @@ def test_subversion__get_url_rev_options():
 def test_get_git_version():
     git_version = Git().get_git_version()
     assert git_version >= parse_version('1.0.0')
+
+
+@pytest.mark.svn
+def test_subversion__get_vcs_version():
+    """
+    Test Subversion.get_vcs_version() against local ``svn``.
+    """
+    version = Subversion().get_vcs_version()
+    assert len(version) == 3
+    assert version[0] >= 1
+
+
+@pytest.mark.parametrize('svn_output, expected_version', [
+    ('svn, version 1.10.3 (r1842928)\n'
+     '   compiled Feb 25 2019, 14:20:39 on x86_64-apple-darwin17.0.0',
+     (1, 10, 3)),
+    ('svn, version 1.9.7 (r1800392)\n'
+     '   compiled Mar 28 2018, 08:49:13 on x86_64-pc-linux-gnu',
+     (1, 9, 7)),
```

Comment:
Some other test cases to include: a string that doesn't begin with "svn, version" (e.g. `svn 1.9.7`), a version with only two numeric parts, a version with only one numeric part, and also the empty string.